sklearn/metrics/tests/test_common.py

```diff
@@ -103,7 +96,13 @@
     "accuracy_score": accuracy_score,
     "balanced_accuracy_score": balanced_accuracy_score,
     "unnormalized_accuracy_score": partial(accuracy_score, normalize=False),
-    "confusion_matrix": confusion_matrix,
+
+    "unnormalized_confusion_matrix": confusion_matrix,
```

Comment:
I think it's a bit weird to name them this way, given that ``confusion_matrix`` is unnormalized for us. I'd rather call this one ``confusion_matrix`` and the other one ``normalized_confusion_matrix``.

Reply:
Agreed. It felt very counter-intuitive to name `confusion_matrix` that way.
However, this was necessary to have the scaling test schedule the correct metric.
See https://github.com/dmohns/scikit-learn/blob/94b9bd0d52ee55478ea7607cfd3037c83e4c2f4b/sklearn/metrics/tests/test_common.py#L1009

For the scope of this PR I didn't want to touch any of the actual test code. 

The other option would be to introduce a new list of "unnormalized" metrics and 
use that for scheduling rather than `name.startswith`.

Thoughts?

Reply:
Basically the ones that can be called with a `normalize` which defaults to `True` parameter. Currently this affects `balanced_accuracy_score`, `jaccard_similarity_score`, `zero_one_loss` and `log_loss`. Let me add a comment on why I am doing what I am doing there and see if it gets clearer.